client/js/main.ts

```diff
@@ -5,6 +5,13 @@ import store from './store/store';
 import router from './router';
 Vue.config.productionTip = false;
 
+// TODO: Think through this, move to relevant location (post Vue 3 since it changes the TS implementation)
+interface CuttleWindow extends Window {
+  Cypress: Object;
+  app: Object;
+}
+declare let window: CuttleWindow;
```

Comment:
Converted main.js to main.ts for the initial support and there was an error in here for the window Object so I extended it to add your new props. I think we should figure out where to put these interfaces and then talk about a new shape for the window object.

I was thinking about something like this:
```
window: {
    cuttle: {
        app: {} // only in Cypress
        version: '3.x.x`, // on all builds
        // ...?
    },
}
```